### DISCUSSION TITLE
Error while creating the MD submission file "Merging is blocked"

### DISCUSSION THREAD

**Discussion by Garcia-INPE** [cu-esiil-edu/2024-data-short-course#101] [General]
When I tried to commit the error occurs, same as Gaurav.

@nquarder @eculler , please, help!

**Comment by nquarder** [comment]
Hi @Garcia-INPE and @gauravchakrabarty - We have the settings to that only @eculler and me are able to merge the pull requests for this repository. We'll take a look here today and will merge your work shortly. Thanks again for all your effort for the short course! We'll be in touch as soon as we have the badges ready to distribute (next week hopefully!). :smile: -NQ
